requirements-lint.txt

```diff
@@ -1,13 +1,13 @@
 -r requirements.txt
 argcomplete==2.0.0
-black==23.10.1
+black==23.11.0
 commitizen==3.12.0
 flake8==6.1.0
 isort==5.12.0
-mypy==1.6.1
+mypy==1.7.0
 pylint==3.0.2
 pytest==7.4.3
-responses==0.24.0
+responses==0.24.1
 types-PyYAML==6.0.12.12
 types-requests==2.31.0.10
-types-setuptools==68.2.0.0
+types-setuptools==68.2.0.1
```

Comment:
I guess these dependency changes were added here by mistake, but since it's not too drastic and it's green, let's keep this.